tests/hap/test_svm_hrcsbc.py

```diff
@@ -198,32 +201,43 @@ def test_svm_empty_cats(gather_output_data):
     assert len(bad_tables) == 0, f"Catalog file(s) {bad_tables} is/are unexpectedly empty"
 
 
-@pytest.mark.skip
-def test_svm_point_cats(gather_output_data):
-    # Check that the point catalogs have the expected number of sources
-    cat_files = [files for files in gather_output_data if files.lower().endswith("point-cat.ecsv")]
+# Due to the way the catalogs are filtered, check the size of the total catalog and one of the filter
+# catalogs separately.  The total catalog has the row removed for each source where the constituent 
+# filter catalogs *ALL* have flag>5 for the source.  Rows are NOT removed from the filter table based on
+# flag values.
+def test_svm_point_total_cat(gather_output_data):
+    # Check the output catalogs should contain the correct number of sources -- allows for a broad tolerance
+    print("\ntest_svm_point_total_cat.")
+    tdp_files = [files for files in gather_output_data if files.lower().find("total") > -1 and files.lower().endswith("point-cat.ecsv")]
 
-    num_sources = {cat:len(ascii.read(cat, format="ecsv")) for cat in cat_files}
+    num_sources = {tdp:len(ascii.read(tdp, format="ecsv")) for tdp in tdp_files}
     valid_cats = {}
-    for cat in expected_point_sources.keys():
-        for file in cat_files:
-            if cat in file and "total" in file:
-                valid_cats[cat] = (np.isclose(num_sources[file], expected_point_sources[cat], rtol=0.1), num_sources[file])
+    for tdp in expected_total_point_sources.keys():
+        for file in tdp_files:
+            if tdp in file:
+                tol_limit = tolerance * expected_total_point_sources[tdp]
```

Comment:
It would be more efficient to simply rely on numpy's built-in comparison code to do this; specifically, np.isclose.  The 'tolerance' specified here corresponds to the 'rtol' parameter for 'np.isclose' or perhaps simply use 'tol_limit' as the value for the 'atol' parameter instead (may be more explicit).  Just an observation...